CHANGELOG.md

```diff
@@ -158,8 +167,6 @@ BabbageEraTxBody --> AlonzoEraTxBody --> ....
       - `FromAlonzoUtxoFail` -> `AlonzoInBabbageUtxoPredFailure`
     - `BabbageUtxowPred` -> `BabbageUtxowPredFailure` and constructor:
       - `FromAlonzoUtxowFail` -> `AlonzoInBabbageUtxowPredFailure`
-- Changed `mint` field type to `MultiAsset (Crypto era)` in `MATxBody`, `AlonzoTxBody`, `BabbageTxBody`
```

Comment:
Oops, sorry about that

Reply:
No worries :) You did it right. It was after your PR was merged that the header was just switched from `### Unreleased` to `### 1.1.x`. Not your fault at all.